### ISSUE TITLE
*.py{} in .gitignore is most likely not a used file extension

### ISSUE THREAD

**Issue by hwalinga** [biopython/biopython#2335]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```python
>>> import sys; print(sys.version)
3.5.3 (default, Sep 27 2018, 17:25:39)
[GCC 6.3.0 20170516]
>>> import platform; print(platform.python_implementation()); print(platform.platform())
CPython
Linux-4.19.0-0.bpo.6-amd64-x86_64-with-LinuxMint-3-cindy
>>> import Bio; print(Bio.__version__)
1.74
```

#### The problem

In the `.gitignore` of this repo there is the pattern `*.py{}`. I thought this was a strange pattern and when I asked on [Stackoverflow](https://stackoverflow.com/questions/58753064/what-is-the-py-extension) nobody knew what it was and just considered it a bug. 

An unused pattern in a `.gitignore` file isn't that big of a problem, but I was working on this repo and I often use `ripgrep` to quickly find stuff in files. One of the features of `ripgrep` is that when used in a `git` repo, it ignores files in the `.gitignore` file. However, ripgrep interprets the `{}` in the `.gitignore` (as a glob brace expansion) causing ripgrep to exclude all the `*.py` files in the repo, making it useless for me to use. 

So, for my workflow (and others using `ripgrep`) I would appreciate it if this line was just removed, as it is unlikely this is really a file extension. Or maybe someone actually knows what program produces files with the file extension `.py{}`. (git blame shows it was introduces by @peterjc )

**Comment by peterjc** [comment]
It was added in 4c7c62939426aee65e4ad69979515817e9a3091b and according to my commit comment, I thought these extensions were from PyPy - which has come a long way in 5 years and may not produce  such files anymore. Or, looking back, it might have been from Jython or IronPython - neither of which we officially support?

Do you use PyPy? If it doesn't produce the files, then it should be safe to update the ``.gitignore`` file.

**Comment by hwalinga** [comment]
Yes, I can remove this entry safely and be happy using `ripgrep` again. But raising this issue was more intended to remove this from the master here so that any people after me using `ripgrep` while working on this repo won't encounter the same issue, as it is rather hard to figure out on your own why `ripgrep` does not work, and it is unlikely this is really a file extension that is in use.

**Comment by peterjc** [comment]
If someone using PyPy could confirm it doesn’t make (or no longer makes)
these files, then a pull request updating the .gitignore would be great.

**Comment by hwalinga** [comment]
Good. I compiled a project with pypy and it did not create such files.

**Comment by peterjc** [comment]
Great - do you want to make a pull request?

**Comment by peterjc** [comment]
Closed by #2345
